internal/processor/otel/logs.go

```diff
@@ -120,12 +128,56 @@ func (c *Consumer) convertLogRecord(
 		}
 		event.Span.ID = spanID.HexString()
 	}
-	if attrs := record.Attributes(); attrs.Len() > 0 {
+	attrs := record.Attributes()
+	if isError(record) {
+		event.Processor = model.ErrorProcessor
+		setErrorLabels(attrs, &event)
+		updateErrorType(attrs, &event)
+	} else {
 		setLabels(attrs, &event)
 	}
 	return event
 }
 
+func updateErrorType(m pcommon.Map, event *model.APMEvent) {
+	if event.Error == nil {
+		event.Error = &model.Error{}
+	}
+
+	event.Error.Type = "crash"
```

Comment:
Setting the `event.Error.Type = "crash"` is mobile-specific, but this logic here is generic for all OTel errors / exceptions. 

Instead, we should set the `event.Error.Type` to the value of the `event.name` OTel attribute.

Reply:
Good catch!